packages/contracts/contracts/interfaces/v0.8.x/IFilteredSharedHolder.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Thoughts on calling this `ISharedMinterHolderV0.sol`?

Reply:
sgtm will update to reflect